### DISCUSSION TITLE
Can I choose a project template from a Git Provider different from the one used by my project?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#333] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.7.1

### Description

Hi!

Suppose I have a company with 2 Git Providers X and Y.
Can I create a project that starts from a project template that is on X but the project uses Git Provider Y?

If this is not possible, in the project creation process the available templates should be filtered based on the selected Git Provider.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, when defining the template datamodel (currently only available within the CMS) you can setup a specific `providerId` for that template. By doing so you can have different git providers in your company with different templates available from them; all you have to do is map the `providerId` accordingly.

Please note that if the template `providerId` is not set, the Console will try to download the template with the same `providerId` of the project that is going to be created (this is mostly done for backward compatibitliy)
